code/modules/clothing/head/helmet.dm

```diff
@@ -383,6 +383,13 @@
 	anti_hug = 50
 	resistance_flags = UNACIDABLE
 
+/obj/item/clothing/head/helmet/marine/veteran/pmc/commando/heavy
+	name = "\improper PMC MX-H Cathaphract commando helmet"
+	desc = "A fully enclosed, armored helmet with hardened cameras, made for Nanotrasen elite commandos."
+	icon = 'icons/mob/clothing/headwear/ert_headwear.dmi'
+	icon_state = "commando_helmet_heavy"
+	hard_armor = list(MELEE = 20, BULLET = 10, LASER = 15, ENERGY = 15, BOMB = 25, BIO = 50, FIRE = 50, ACID = 50)
```

Comment:
Why do they need armour even better than immunity to almost every source of damage in the game?